public/js/form.js

```diff
@@ -70,6 +74,18 @@ export class Form extends Component {
                 this.query.current.value(data['query']);
             }
 
+            setTimeout(function () {
+                const checkboxDatabases = $('.checkbox-database');
+
+                if (data['preSelectedDbs']) {
+                    const preselectData = data['preSelectedDbs'];
+                    $.each(preselectData, function(index, db) {
+                        const matchingCheckbox = checkboxDatabases.filter(`[value="${db.id}"]`);
+                        matchingCheckbox.prop('checked', true);
+                    });
+                }
+            }, 0);
+
```

Comment:
Any reason we can't use React here? We've been trying to reduce dependency on jQuery in general as well as just using React for dom manipulation.